apps/safe-claiming-app/src/components/steps/Claim/index.tsx

```diff
@@ -236,7 +236,7 @@ const Claim = ({ handleBack, state, handleUpdateState, handleNext }: Props) => {
                   variant="contained"
                   size="large"
                   disableElevation
-                  sx={{ width: 1, paddingX: 0 }}
+                  sx={{ width: 1, paddingX: 0, height: "47px" }}
```

Comment:
I don't know whats with the buttons in the claiming app.
I would change it by doing the following:

1. remove `size="large"`
2. fix the styles in the theme.ts (currently the buttons are way too small):
This seems to work:
```ts
      MuiButton: {
        styleOverrides: {
          sizeLarge: {
            fontSize: "16px",
            padding: "12px 40px",
          },
          sizeMedium: {
            fontSize: "16px",
            padding: "12px 24px",
          },
          root: ({ theme }) => ({
            borderRadius: theme.shape.borderRadius,
            fontWeight: "bold",
            lineHeight: 1.25,
            borderColor: theme.palette.primary.main,
            textTransform: "none",
            "&:hover": {
              boxShadow: "none",
            },
          }),
        },
      },
```

3. customize the padding to slightly smaller so that the button text does not wrap (8px for instance works for me)

Reply:
> I don't know whats with the buttons in the claiming app.

In this case, the button was hard to align with the TextField input given it always preserves the input field error height.

I am going to try your suggestion to see if we get to the same result 🙏

Reply:
Why does the button have to be the same size? Can't it be just centered vertically?

Reply:
Hmm, after I remove the `size` property the button still not quite adjusts to the input height.
I'd go more pragmatic in this case and simply pass it the height the way it is. WDYT?